### ISSUE TITLE
Addresscheck - undefined index if non-covered country is used

### ISSUE THREAD

**Issue by berwa** [PAYONE-GmbH/magento-2#343]
Hello Payone-team,
today we ran into an issue when address-check is enabled, but the address' country is not covered by the check:
When trying to complete the final checkout step with a billing-address in Switzerland and a shipping-address in Japan an empty alert-box is displayed and it is not possible to complete the checkout.

The following entry is added to Magento exception.log: 
```
Notice: Undefined index: status in .../vendor/payone-gmbh/magento-2/Service/V1/Addresscheck.php on line 142
```

Environment:
* Magento-version: Commerce 2.3.5-p2
* Module-version: Release 3.1.0
* allowed countries: all (worldwide)

Configurations of _Address checking_ section of Payone-configuration:
* Enabled: Yes
* Mode: Test
* Check Billing Address: Basic
* Check Shipping Address: Basic
* Confirm Address correction: Yes

From what we found out so far Japan is not contained in the list of supported countries for the address-check (https://github.com/PAYONE-GmbH/magento-2/blob/master/Model/Api/Request/Addresscheck.php#L39) which results in only a 'wrongCountry'-entry being added to the array (https://github.com/PAYONE-GmbH/magento-2/blob/master/Model/Api/Request/Addresscheck.php#L151), but the implementation of the Service-class expects a 'status' entry in that array (https://github.com/PAYONE-GmbH/magento-2/blob/master/Service/V1/Addresscheck.php#L142) which causes the above mentioned notice.

Our idea was to handle that case the same way it is done for a disabled check (https://github.com/PAYONE-GmbH/magento-2/blob/master/Model/Api/Request/Addresscheck.php#L146).

```
if (!$this->validateTypeToCountry($sType, $oAddress)) {
            return true;
}
```

A quick test confirmed that this would resolve the issue, and does not brake the validation for supported countries (e.g. Germany); but we are not sure if this would cause any trouble somewhere else.

Thank you in advance and Best regards

**Comment by hreinberger** [comment]
Hi @berwa 
thanks for your thorough description. We'll look into this and come up with a fix.

**Comment by hreinberger** [comment]
fixed in #347
